### PR TITLE
Fix is_antenatal checkbox display

### LINKED ISSUE #3214: antenatal malfunction : Patient Form
image
image

when we create a antenatal patient, and click update detail button, the radio button doesn't seem selected

### PR DESCRIPTION
**Pull request by Ashesh3**
Fixes #3214 

image

### CODE REVIEW

**Review by kunatastic — APPROVED** [review]

File: src/Components/Patient/PatientRegister.tsx [hunk 1]
```diff
@@ -375,7 +375,7 @@ export const PatientRegister = (props: PatientRegisterProps) => {
             village: res.data.village ? res.data.village : "",
             medical_history: [],
             is_antenatal: res.data.is_antenatal
-              ? res.data.is_antenatal
+              ? String(res.data.is_antenatal)
```
Comment (developedBySJ) [inline src/Components/Patient/PatientRegister.tsx]: Use `is_antenatal: String(!!res.data.is_antenatal)`. It looks more clean and readable.

**Review by developedBySJ — APPROVED** [review]

File: src/Components/Patient/PatientRegister.tsx (hunk 1 above)
Comment (Ashesh3) [reply src/Components/Patient/PatientRegister.tsx]: Hey, thanks for the suggestion. Would you recommend me changing all other occurrences of such typecasting too (In this file)? 
image
I used `String(` to maintain consistency with the existing codebase but would be happy to change it everywhere in the file, as per your suggestion, if you believe that's the better approach.

**Comment by nihal467** [comment]
@Ashesh3 @developedBySJ once the work is completed, can the status to need to test

**Comment by nihal467** [comment]
@Ashesh3 https://deploy-preview-3215--care-net.netlify.app/facility/bd466cf6-0fde-46b1-9591-0acf9d955dd1/patient/8027c51f-f6bf-449f-81a2-1c1fba999ac0

this patient was a female patient, which i updated sex to male later, but the antenatal badge is still there and it is not removed, can you check that

**Comment by Ashesh3** [comment]
@nihal467 Fixed it so that antenatal badge and details are only displayed on the Patient detail page when the sex is female.

**Comment by nihal467** [comment]
@Ashesh3 test failed
image

it is showing in the patient tab

https://deploy-preview-3215--care-net.netlify.app/facility/bd466cf6-0fde-46b1-9591-0acf9d955dd1/patient/8027c51f-f6bf-449f-81a2-1c1fba999ac0

**Comment by Ashesh3** [comment]
> @Ashesh3 test failed image
> 
> it is showing in the patient tab
> 
> https://deploy-preview-3215--care-net.netlify.app/facility/bd466cf6-0fde-46b1-9591-0acf9d955dd1/patient/8027c51f-f6bf-449f-81a2-1c1fba999ac0

Should be fixed now.

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**